nautobot/extras/jobs.py

```diff
@@ -486,6 +520,15 @@ def as_form(cls, data=None, files=None, initial=None, approval_view=False):
             label="Profile job execution",
             help_text="Profiles the job execution using cProfile and outputs a report to /tmp/",
         )
+        # It is not clear to me why pylint thinks that this is a constant, the comparison and form rendering
+        # works correctly.
+        if cls.is_singleton:  # pylint: disable=using-constant-test
+            form.fields["_ignore_singleton"] = forms.BooleanField(
```

Comment:
Just checking - I see a mix of `_ignore_singleton` and `_ignore_singleton_lock` in this code, and it's not immediately obvious to me where one gets translated to the other. Should it be `ignore_singleton_lock` throughout?